kmtomiles/main.py

```diff
@@ -0,0 +1,23 @@
+import discord
+from discord.ext import commands
+
+CONST_prefix = '!'
```

Comment:
`CONST_prefix` appears to be labelled as a constant, so it should be just `PREFIX` in accordance with [PEP-8](https://peps.python.org/pep-0008/#constants)

Reply:
`bot_TOKEN` appears to be constant too, and so should also be `BOT_TOKEN`